src/status_im/data_store/realm/schemas/base/core.cljs

```diff
@@ -64,6 +64,8 @@
           extension/v12
           account/v15])
 
+(def v17 v15)
```

Comment:
could you elaborate?

Reply:
Kept the schema same. Migration only affected the data in the old realm, not its structure.

Reply:
thanks

Reply:
ok but why go back to v15 and not v16 ?

Reply:
Oh that, good catch, merge mistake.